tests/end2end/features/downloads.feature

```diff
@@ -126,6 +126,14 @@ Feature: Downloading things from a website.
         Then "cancelled" should be logged
         And "cancelled" should be logged
 
+    # https://github.com/The-Compiler/qutebrowser/issues/1535
+    Scenario: Cancelling a MHTML download (issue 1535)
+        When I open data/downloads/issue1535.html
+        And I run :download --mhtml
+        And I wait 1s
```

Comment:
What's the reason for waiting here? I think the test would work fine without that, as at the time the `:download` handler returns there definitely should be a download running, no?